package.json

```diff
@@ -17,7 +17,9 @@
     "coverage": "npm run unit -- --cov --coverage-report=html",
     "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
     "bench": "branchcmp -r 2 -g -s \"npm run benchmark\"",
+    "bench:windows": "branchcmp -r 2 -g -s \"npm run benchmark:windows\"",
```

Comment:
Why do we need separate scripts for windows?

Reply:
See `benchmark:windows`. When there is `npx` before `concurrently` on Windows, for some reason an error is thrown:
```
> npx concurrently -k -s first "node ./examples/simple.js" "npx autocannon -c 100 -d 5 -p 10 localhost:3000/"

10 '.' is not recognized as an internal or external command,
10 operable program or batch file.
10 The filename, directory name, or volume label syntax is incorrect.
10 ./examples/simple.js exited with code 1
```

If you have any ideas how to make script universally applicable (maybe some additional escaping is needed somewhere?..), that would be much better, of course.

Reply:
Could you try

```
npx concurrently -k -s first "node './examples/simple.js'" "npx autocannon -c 100 -d 5 -p 10 localhost:3000/"
```

Reply:
Same thing:
```
> npx concurrently -k -s first "node './examples/simple.js'" "npx autocannon -c 100 -d 5 -p 10 localhost:3000/"

10 '.' is not recognized as an internal or external command,
10 operable program or batch file.
10 The filename, directory name, or volume label syntax is incorrect.
```

. itself doesn't seem to be a problem, if I remove it, there is still an error:
```
> npx concurrently -k -s first "node 'examples/simple.js'" "npx autocannon -c 100 -d 5 -p 10 localhost:3000/"

10 'examples' is not recognized as an internal or external command,
10 operable program or batch file.
10 The filename, directory name, or volume label syntax is incorrect.
```

Reply:
@StarpTech Do you have any other ideas, or should we keep it as-is?

Reply:
This works on my win10 with git bash (removed the apostrophe `'`)
```
npx concurrently -k -s first "node ./examples/simple.js" "npx autocannon -c 100 -d 5 -p 10 localhost:3000/"
```

Reply:
It still doesn't for me for some reason:
![image](https://user-images.githubusercontent.com/1847934/84655793-46cbe700-af1a-11ea-90e9-e90d1438e4ea.png)

Reply:
is the `npx` command working alone?